### PR TITLE
Cleanup changeset entries

### PR DESCRIPTION
**Pull request by kyledurand**
### WHY are these changes introduced?

Fixes https://github.com/Shopify/web/issues/105446

This is a first pass at general cleanup. Let me know if there's anything else you want addressed in here @chloerice @sophschneider

### CODE REVIEW

File: .changeset/curly-chairs-speak.md [hunk 1]
```diff
@@ -1,5 +0,0 @@
----
-'@shopify/polaris': minor
----
-
-Add a search field to filter ActionList that have more than 10 items
```
Comment (kyledurand) [inline .changeset/curly-chairs-speak.md:5]: @sam-b-rose GH had you as the author of this file but I think we already have an entry for it in main.

**Review by sophschneider — APPROVED** [review]
🙌

File: .changeset/fuzzy-pears-jog.md [hunk 2]
```diff
@@ -2,4 +2,4 @@
 '@shopify/polaris': major
 ---
 
-Updated `borderRadius` props to match web spec
+Updated `borderRadius` props on `Box` to match locical property naming conventions.
```
Comment (chloerice) [inline .changeset/fuzzy-pears-jog.md]:
```suggestion
Updated `borderRadius` props on `Box` to match logical property naming conventions.
```

File: .changeset/honest-stingrays-act.md [hunk 3]
```diff
@@ -2,4 +2,4 @@
 '@shopify/polaris': major
 ---
 
-Removed connectedDislosure prop on button
+Removed `connectedDislosure` prop on `button`
```
Comment (chloerice) [inline .changeset/honest-stingrays-act.md]:
```suggestion
Removed `connectedDisclosure` prop from `Button`
```

**Review by chloerice — APPROVED** [review]

**Pull request merged**